### PR TITLE
Ft mark collected art

### PR DESCRIPTION
**Pull request by OloruntobiAwoderu**
# Description
Mark Art as collected
Add attribution link to Readme and code

## Type of change
An EndPoint to MarkArt as collected
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: api/controllers/art.js [hunk 1]
```diff
@@ -23,21 +55,17 @@ module.exports = {
...
+      const schoolOrders = await models.order
+        .find({ schoolId: id, status: "pending" })
+        .populate("transactionId")
+        .populate("buyerId")
+        .exec();
       return successResponse(res, 200, schoolOrders);
     } catch (error) {
-      return next(error)
+      return next(error);
     }
```
Comment (iamcristos) [inline api/controllers/art.js]: I think you need to pull from developement. This method is not updated

Comment (OloruntobiAwoderu) [reply api/controllers/art.js]: done

File: models/product.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-      required: [true, 'Product must have a name'],
+      required: [true, "Product must have a name"],
+      trim: true
+    },
+    artistName: {
+      type: String,
+      required: [true, "Product must have an Artist name"],
       trim: true
     },
     medium: {
```
Comment (petrussola) [inline models/product.js]: As per conversation with Frontend, I think medium should be compulsory field now and maybe an enum. @iamcristos and @ruairidhflint can you confirm? @ruairidhflint which fields should we include?

Comment (OloruntobiAwoderu) [reply models/product.js]: All those can be updated in subsequent PR'S

Comment (ruairidhflint) [reply models/product.js]: I think medium should be compulsory but I guess for now it's not essential...we can get it working and then decide on a rigid structure for a subsequent PR. 

It seems like a lot of require fields but given that schools are trying to sell things, I think it is their best interest to supply as much information as possible.

**Pull request merged**